types/node/index.d.ts

```diff
@@ -22,7 +22,6 @@
 //                 Mariusz Wiktorczyk <https://github.com/mwiktorczyk>
 //                 Mohsen Azimi <https://github.com/mohsen1>
 //                 Nicolas Even <https://github.com/n-e>
-//                 Nicolas Voigt <https://github.com/octo-sniffle>
```

Comment:
Why this change?

Reply:
@SimonSchick The account doesn't exist anymore. I noticed because it's not properly linked in @typescript-bot's comments.

Reply:
Ok, in the future consider adding a note for unrelated changes such as these as it reduces roundtrip time.